SpatialGDK/Source/SpatialGDK/Public/SpatialView/OpList/EntityComponentOpList.h

```diff
@@ -18,6 +18,8 @@ struct EntityComponentOpListData : OpListData
 	TArray<ComponentData> DataStorage;
 	TArray<ComponentUpdate> UpdateStorage;
 	TArray<StringStorage> MessageStorage;
+	TArray<TArray<Worker_Entity>> QueriedEntities;
+	TArray<TArray<Worker_ComponentData>> QueriedComponents;
```

Comment:
These aren't owning types. They should probably be just TArray<TArray<ComponentData>>. Everything else doesn't need storage in the class type.